homeassistant/components/emulated_hue/__init__.py

```diff
@@ -156,7 +158,6 @@ async def start_emulated_hue_bridge(event):
         nonlocal protocol
         nonlocal site
         nonlocal runner
-        await config.async_setup()
```

Comment:
This should stay in

Reply:
I based the change on https://github.com/home-assistant/core/pull/52980#discussion_r671751529

Reply:
We still need it but can move it above. suggestion above